docs/rules/eqeqeq.md

```diff
@@ -67,6 +67,13 @@ foo === null
 
 ```
 
+This rule optionally takes a second argument which should be an object with the following supported properties:
```

Comment:
Suggest adding a comma after "second argument"- feels like that might read better.